### ISSUE TITLE
Allow opening local files

### ISSUE THREAD

**Issue by hkuhn42** [arx-8/swagger-viewer#83]
Often, swagger files are not hosted on github. Would be nice to just enable the viewer for other locations like local disc.

**Comment by arx-8** [comment]
Hi, @hkuhn42. Sorry for the very late reply, and Thank you for your great idea!
When reading and writing Swagger locally, I recommend the following tools:

- Swagger Editor (Web app): https://editor.swagger.io
- Swagger Viewer (VSCode Extension): https://marketplace.visualstudio.com/items?itemName=Arjun.swagger-viewer

Do these meet your demand? If not, could you please provide more details about your needs?

I feel the above tools are more useful than my extension, especially when it comes to managing local Swagger.
In other words, I can't think of a strong reason to reinvent the above tools and don't know what to develop.
